### PR TITLE
Clarifying metrics for resources estimator

### PR DESCRIPTION
**Pull request by bettinaheim**

### CODE REVIEW

**Comment by bettinaheim** [comment]
@SyntaxC4 could you review, given that @KittyYeungQ is on vacation?

File: articles/user-guide/machines/resources-estimator.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ QubitCount      2       2
 BorrowedWidth   0       0
 ```
 
+For more information about the collected metrics, see the description of the [Reported Metrics](#metrics-reported). The default configuration used for the `ResourcesEstimator` counts only `T` gates. Other configurations can be set by creating and customizing a [`QCTraceSimulator`](xref:microsoft.quantum.machines.overview.qc-trace-simulator.intro) instance.
```
Comment (viktorveis) [inline articles/user-guide/machines/resources-estimator.md:63]: `counts only T gates` can benefit from clarifying that this statement is specific to the Depth counter.

Comment (viktorveis) [inline articles/user-guide/machines/resources-estimator.md:63]: Consider moving the sentence about QCTraceSimulator to the Metrics section that talks about Depth.

**Comment by KittyYeungQ** [comment]
@cjgronlund @SoniaLopezBravo Could you review and provide Bettina permission to assign reviewers in this repo? Thank you.

**Review by SoniaLopezBravo — APPROVED** [review]

File: articles/user-guide/machines/resources-estimator.md (hunk 1 above)
Comment (SoniaLopezBravo) [reply articles/user-guide/machines/resources-estimator.md:63]: True, but it is actually stated a few lines above, in the table of metrics.

**Pull request merged**